### PR TITLE
Change testnet url to our own stellar deployment

### PR DESCRIPTION
**Pull request by DylanVerstraete**
related to: https://github.com/threefoldtech/tf_operations/issues/22

### CODE REVIEW

**Review by m-motawea — CHANGES_REQUESTED** [review]
Please use a proper domain name

**Comment by DylanVerstraete** [comment]
> Please use a proper domain name

@hossnys can you maybe look to give this a proper domain name?

**Comment by waleedhammam** [comment]
I tried to test the branch with test wallet, it fails in wallet adding trustlines and transactions, any idea ?

```
stellar_sdk.exceptions.BadResponseError: {
  "type": "https://stellar.org/horizon-errors/timeout",
  "title": "Timeout",
  "status": 504,
  "detail": "Your request timed out before completing.  Please try your request again. If you are submitting a transaction make sure you are sending exactly the same transaction (with the same sequence number)."
}

{
  "type": "https://stellar.org/horizon-errors/timeout",
  "title": "Timeout",
  "status": 504,
  "detail": "Your request timed out before completing.  Please try your request again. If you are submitting a transaction make sure you are sending exactly the same transaction (with the same sequence number)."
}
```

**Comment by DylanVerstraete** [comment]
@waleedhammam can you check if your address exists? 
`curl http://157.245.131.96:8000/accounts/YOURADDRESS`

**Comment by waleedhammam** [comment]
> @waleedhammam can you check if your address exists?
> `curl http://157.245.131.96:8000/accounts/YOURADDRESS`

@DylanVerstraete no i can't `http://157.245.131.96:8000/GCZI5MS7S5DNS6ASZSRYFA4HU7HCCHT2YQNY6EFKOSFJBXNCJWMDNDEB`

**Comment by DylanVerstraete** [comment]
> > @waleedhammam can you check if your address exists?
> > `curl http://157.245.131.96:8000/accounts/YOURADDRESS`
> 
> @DylanVerstraete no i can't `http://157.245.131.96:8000/GCZI5MS7S5DNS6ASZSRYFA4HU7HCCHT2YQNY6EFKOSFJBXNCJWMDNDEB`

`curl http://157.245.131.96:8000/accounts/GCZI5MS7S5DNS6ASZSRYFA4HU7HCCHT2YQNY6EFKOSFJBXNCJWMDNDE`
Url is wrong

**Comment by waleedhammam** [comment]
> > > @waleedhammam can you check if your address exists?
> > > `curl http://157.245.131.96:8000/accounts/YOURADDRESS`
> > 
> > 
> > @DylanVerstraete no i can't `http://157.245.131.96:8000/GCZI5MS7S5DNS6ASZSRYFA4HU7HCCHT2YQNY6EFKOSFJBXNCJWMDNDEB`
> 
> curl http://157.245.131.96:8000/accounts/GCZI5MS7S5DNS6ASZSRYFA4HU7HCCHT2YQNY6EFKOSFJBXNCJWMDNDE
> Url is wrong

can you try with a new wallet ?, I created one, activated but failed to find the wallet

**Comment by hossnys** [comment]
> > Please use a proper domain name
> 
> @hossnys can you maybe look to give this a proper domain name?

record added here http://stellar-watcher.grid.tf/

**Comment by DylanVerstraete** [comment]
Can we test and verify that the url is indeed set and all the operations work as they are supposed to?

- transfer
- trustlines
- activation
- ..

File: jumpscale/clients/stellar/stellar.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
 
 _THREEFOLDFOUNDATION_TFTSTELLAR_SERVICES = {"TEST": "testnet.threefold.io", "STD": "tokenservices.threefold.io"}
-_HORIZON_NETWORKS = {"TEST": "https://horizon-testnet.stellar.org", "STD": "https://horizon.stellar.org"}
+_HORIZON_NETWORKS = {"TEST": "http://stellar-watcher.grid.tf", "STD": "https://horizon.stellar.org"}
```
Comment (m-motawea) [inline jumpscale/clients/stellar/stellar.py:23]: shouldn't the domain name include an indication of testnet? also shouldn't it be https?

Comment (hossnys) [reply jumpscale/clients/stellar/stellar.py]: updated here: c86762c

**Comment by m-motawea** [comment]
I could create a funded wallet from the dashboard and could create and extend pools using the created wallet.

**Review by m-motawea — APPROVED** [review]

**Review by waleedhammam — APPROVED** [review]

**Comment by m-motawea** [comment]
@DylanVerstraete is testnet explorer using this node now?

**Comment by DylanVerstraete** [comment]
> @DylanVerstraete is testnet explorer using this node now?

No, not yet. Does not make any difference for this PR though. We will implement that this week.

**Comment by DylanVerstraete** [comment]
Converted to draft in favor of restoring testnet first. This can later be reopened in the next release

**Pull request closed without merging**